students/11th/taehyunlee/westagram/User/urls.py

```diff
@@ -0,0 +1,8 @@
+from django.urls import path
+
+from . import views
+
+urlpatterns = [
+    path('SignUp', views.SignUp.as_view()),
+    path('SignIn', views.SignIn.as_view()),
```

Comment:
엔드포인트 주소는 Camel case로 작성하지는 않습니다. 수정 부탁 드립니다.
```suggestion
    path('sign-up', views.SignUpView.as_view()),
    path('sign-in', views.SignInView.as_view()),
```